src/test/scala/chiselTests/experimental/DataMirrorSpec.scala

```diff
@@ -55,4 +74,18 @@ class DataMirrorSpec extends ChiselFlatSpec {
     }
     ChiselStage.elaborate(new MyModule)
   }
+
+  it should "support getParent for normal modules" in {
+    ChiselStage.elaborate(new Parent)
+  }
+
+  it should "support getParent for normal modules even when used in a D/I context" in {
+    import chisel3.experimental.hierarchy._
+    class Top extends Module {
+      val defn = Definition(new Parent)
+      val inst = Instance(defn)
```

Comment:
what happens in `DataMirror.getParent(inst) should be(this)`?

Reply:
Compiler error 😇, see https://github.com/chipsalliance/chisel3/pull/2825#discussion_r1014729172.